data-raw/healthy-lives/behavioural-risk-factors/drug-misuse.R

```diff
@@ -0,0 +1,71 @@
+# ---- Load packages ----
+library(tidyverse)
+library(readxl)
+library(httr)
+
+# ---- Scrape URL for drug deaths ----
+# ---- Source: https://www.ons.gov.uk/peoplepopulationandcommunity/birthsdeathsandmarriages/deaths/datasets/drugmisusedeathsbylocalauthority ----
```

Comment:
`# ---- ` Tends to be used to denote headings.  R studio will read it as such.
When you want to make a comment within a heading you can use `#` (without the ----). It makes the script easier to read and also allows you to navigate through the script in sections, rather than everytime you make a comment.